xtask/src/metadata.rs

```diff
@@ -414,6 +409,54 @@ impl runtime_v1::RuntimeV1 for Runtime {
     fn register_node(&mut self, metadata: &Self::Metadata) {
         self.node = Some(metadata.lock().unwrap().clone());
     }
+
+    fn drop_argument_metadata(&mut self, state: Self::ArgumentMetadata) {
+        drop(state);
+    }
+
+    fn drop_metadata(&mut self, state: Self::Metadata) {
+        drop(state);
+    }
+
+    fn drop_tensor_hint(&mut self, state: Self::TensorHint) {
+        drop(state);
+    }
+
+    fn drop_tensor_metadata(&mut self, state: Self::TensorMetadata) {
+        drop(state);
+    }
+
+    fn drop_argument_hint(&mut self, state: Self::ArgumentHint) {
+        drop(state);
+    }
+
```

Comment:
I don't think we need to explicitly `drop()` the `state` variables here. 

Values are automatically dropped whenever they go out of scope, and I'm pretty sure we get a default implementation for each of the `drop_XXX()` methods that does nothing except consume the `state`.

```suggestion
```